client/src/features/app/component.js

```diff
@@ -0,0 +1,16 @@
+import React, { PureComponent } from 'react';
+
+import Button from '../button';
+import MovieList from '../movie-list';
+
+import './component.less'
+
+class App extends PureComponent {
+  render() {
+    return(
+      <MovieList className="movie-list" />
+    )
+  }
+}
```

Comment:
please add ESLint, add `eslint:recommended` and `airbnb` rule sets.
here you will see error, as this component should be `pure function` as there is no state/handlers/other business logic.